packages/clerk-js/src/ui/Components.tsx

```diff
@@ -149,7 +160,21 @@ const Components = (props: ComponentsProps) => {
   const { signInModal, signUpModal, userProfileModal, organizationProfileModal, createOrganizationModal, nodes } =
     state;
 
+  // TODO: rename these
+  const urlClerkState = getClerkQueryParam('__clerk_state') ?? '';
```

Comment:
🙃 Let's convert the `__clerk_state` into a constant and re-use it across the code base.